share/spice/iplookup/iplookup.js

```diff
@@ -30,7 +37,7 @@
             templates: {
                 group: 'text',
                 options: {
-                    content: api_result.ddg.blacklists.length ? Spice.iplookup.content : null,
```

Comment:
@moollaza See https://github.com/duckduckgo/zeroclickinfo-spice/commit/b01cb4cf767688ac6eabc9cc6424f9f28d80cbad 

I was using that, however I found that getProperty would return an empty array `[]` if the object didn't exist. This causes problems with checking the value using the conditional statement. I'll need to test again, however that's why I choose not to use it.